src/Psl/Dict/unique.php

```diff
@@ -16,6 +16,11 @@
  */
 function unique(iterable $iterable): array
 {
+    
+    if (\is_array($iterable)) {
+        return \array_unique($iterable);
+    }
```

Comment:
please import functions instead using `\` ( `use is_array;` and `use array_unique;` )